pkg/lib/interaction/nodes/settings_action_end.go

```diff
@@ -0,0 +1,46 @@
+package nodes
+
+import (
+	"github.com/authgear/authgear-server/pkg/lib/interaction"
+	"github.com/authgear/authgear-server/pkg/lib/oauth/oauthsession"
+)
+
+func init() {
+	interaction.RegisterNode(&NodeSettingsActionEnd{})
+}
+
+type EdgeSettingsActionEnd struct{}
+
+func (e *EdgeSettingsActionEnd) Instantiate(ctx *interaction.Context, graph *interaction.Graph, input interface{}) (interaction.Node, error) {
+	return &NodeSettingsActionEnd{}, nil
+}
+
+type NodeSettingsActionEnd struct{}
+
+func (n *NodeSettingsActionEnd) Prepare(ctx *interaction.Context, graph *interaction.Graph) error {
+	return nil
+}
+
+func (n *NodeSettingsActionEnd) GetEffects() ([]interaction.Effect, error) {
+	return []interaction.Effect{
+		interaction.EffectOnCommit(func(ctx *interaction.Context, graph *interaction.Graph, nodeIndex int) error {
+			entry, err := ctx.OAuthSessions.Get(ctx.OAuthSessionID)
```

Comment:
Should do nothing if OAuthSession is not found (it could happen. We can change password directly in the settings page)